### DISCUSSION TITLE
how to install in windows

### DISCUSSION THREAD

**Discussion by ilivecoding** [github-linguist/linguist#6148] [Q&A]
![图片](https://user-images.githubusercontent.com/7898334/198755029-fe89c9aa-4318-4d8c-b24c-7efd301b2c51.png)

**Comment by lildude** [comment]
I don't understand the language your screenshot is written in, but you would need to install Ruby and then install the gem like your would any other gem on Windows. I don't have access to a Windows system so can't provide details, but there are plenty of links to methods on Google.

**Comment by ilivecoding** [comment]
OK，thanks.

从 Windows 版邮件发送

发件人: Colin Seymour
发送时间: 2022年10月29日 16:44
收件人: github/linguist
抄送: shaw; Author
主题: Re: [github/linguist] how to install in windows (Discussion #6148)

I don't understand the language your screenshot is written in, but you would need to install Ruby and then install the gem like your would any other gem on Windows. I don't have access to a Windows system so can't provide details, but there are plenty of links to methods on Google.
